test/expected.txt

```diff
@@ -1,305 +1,304 @@
-|| exit: Exits the program.
-|| Example: exit
+|| The given file name [ ] is not a valid file name!
+|| Exiting Address Book... Good bye!
 || ===================================================
-|| Enter command: || [Command entered:  delete 1]
-|| The person index provided is invalid
 || ===================================================
-|| Enter command: || [Command entered:  view 1]
-|| The person index provided is invalid
 || ===================================================
-|| Enter command: || [Command entered:  viewall 1]
-|| The person index provided is invalid
 || ===================================================
-|| Enter command: || [Command entered:  clear]
-|| Address book has been cleared!
+|| AddessBook Level 1 - Version 1.0
+|| Welcome to your Address Book!
 || ===================================================
-|| Enter command: || [Command entered:  list]
-|| 
-|| 0 persons listed!
+|| The given file name [directoryThatDoesNotExist/valid.filename] is not a valid file name!
+|| Exiting Address Book... Good bye!
 || ===================================================
-|| Enter command: || [Command entered:  add wrong args wrong args]
-|| Invalid command format! 
-|| add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
-|| Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
 || ===================================================
-|| Enter command: || [Command entered:  add Valid Name 12345 e/valid@email.butNoPhonePrefix a/valid, address]
-|| Invalid command format! 
-|| add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
-|| Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
 || ===================================================
-|| Enter command: || [Command entered:  add Valid Name p/12345 valid@email.butNoPrefix a/valid, address]
-|| Invalid command format! 
-|| add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
-|| Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
 || ===================================================
-|| Enter command: || [Command entered:  add Valid Name p/12345 e/valid@email.butNoAddressPrefix valid, address]
-|| Invalid command format! 
-|| add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
-|| Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
+|| AddessBook Level 1 - Version 1.0
```

Comment:
Do check if all the changes in this document are relevant for this feature's PR.